overview.md

```diff
@@ -395,11 +401,17 @@ that it used to ensure uniqueness. However we know part of the substring that
 the name should have (and have ourselves chosen it to be unique).
 
 The other clauses are simlilar but happen to use different types of predicates.
-The `contains_pred_list()` is checking that each of the predicates holds on a
-particular object. This would be the same as the `jp.DICT_SUBSET` predicate,
-however reporting is different should one of these predicates fail, the individual
-attributes will be shown as opposed to the entire dict since these are independent
-predicates.
+The `contains_match()` is a shorthand notation for specifying a collection of
+object attributes with their validation predicates, all of which must be
+satisifed by the same object. This is functionally equivalent to listing all
+the individual path predicates, but more concise to read and write, and allows
+more concise reporting as well.
+
+Note that the `path_contains_value(path, {key: value})` call in the previous
```

Comment:
I think we should work out how to get the same reporting while using the more complex path keys. Nested dictionaries are hard to reason about -- I'd rather have a flat dictionary with paths pointing to predicates.